trunk/src/lsc/lscastrodef.py

```diff
@@ -69,6 +69,28 @@ def wcsstart(img,CRPIX1='',CRPIX2=''):
         else: CRPIX2= 1000.+CRPIX2
         CDELT1=2
         CDELT2=2
+    elif 'sq' in _instrume:
+        angle=readkey3(hdr,'ROLLERDR')#posang)
+        theta=(angle*pi/180.)
+        pixscale=float(readkey3(hdr,'PIXSCALE'))
+        CDELT0=pixscale/3600.
+        CD1_1=CDELT0*cos(theta)
+        CD1_2=CDELT0*sin(theta)
+        CD2_1=CDELT0*sin(theta)
+        CD2_2=(-1)*CDELT0*cos(theta)
+        if 'SITEID' in hdr:
+            if hdr['SITEID'] in ['elp','tfn','ogg']:
+                CD1_1 = (-1) * CD1_1
+                CD2_2 = (-1) * CD2_2
```

Comment:
Is this actually true? It looks like this came from copying the Sinistro block, but I have no idea if the same random sites have 180º rotated cameras.

Reply:
Stefano said this is northern vs southern hemisphere

Reply:
I actually have no idea. Probably Griffin is right is just a rotation of the camera.

Reply:
I didn't test the southern hemisphere but it did something sensible for tfn when I ran the wcs stage